charts/pega/values-large.yaml

```diff
@@ -278,3 +278,5 @@ installer:
     upgradeType: "in-place"
     # Specify target rules schema for migration and upgrade
```

Comment:
let's clarify in the comments that the process creates there schemas using these names:
# Specify a name for a target rules schema that the upgrade process creates for patches and upgrades.
targetRulesSchema: ""
# Specify a name for a target data schema that the upgrade process creates for patches and upgrades.
targetDataSchema: ""